scp/lib/booking.dart

```diff
@@ -78,7 +80,7 @@ class _BookingState extends State<Booking> {
             child: Text(
               'Confirmation',
               style: TextStyle(
-                  fontSize: queryWidth * 0.065,
+                  fontSize: screenWidth * 0.065,
```

Comment:
Are you sure that fontSize in relation to screenWidth is a better option than using `textScaleFactor`?

Reply:
The Text widget has a property called textScaleFactor and, put simply, the given font size gets multiplied by the textScaleFactor to determine the font size that is actually rendered on the screen in logical pixels. So, if you wanted the text to be 150% of its normal size then you’d set your textScaleFactor to 1.5.
According to me, the textScaleFactor would come in handy while defining layouts for different screens.